client/src/utils/helpers/inputValidators.js

```diff
@@ -0,0 +1,33 @@
+export const isNoErrors = errorsObj => {
+  return Object.values(errorsObj).some(err => err !== '')
+}
```

Comment:
I think there is a wrong logic. 
We could write either: 
`areNoErrors = errorsObj => {
  return Object.values(errorsObj).every(err => err === '')
}`
 or 
`isError = errorsObj => {
  return Object.values(errorsObj).some(err => err !== '')
}` and then
`const inputIsValid = !validate()`

Reply:
Yep, you're right, thanks